### PR TITLE
Edit Fluentd license info so GitHub recognizes it

### PR DESCRIPTION
**Pull request by eirinikos**
Hello! 👋 

(CCing @dankohn who has requested this work so that the license will appear correctly in https://landscape.cncf.io/grouping=landscape&landscape=logging&selected=fluentd)

GitHub uses a library called Licensee to identify a project's license
type. It shows this information in the status bar and via the API if it
can unambiguously identify the license.

This commit removes the COPYING file (which duplicates info that's
already included in LICENSE), so that the license type identification
(as Apache 2.0) will succeed.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2018 FURUHASHI Sadayuki
```
Comment (dankohn) [inline LICENSE]: Should this be `Copyright 2018 Fluentd Authors`?

Cc @edsiper

Comment (eirinikos) [reply LICENSE]: What do you think of using `Copyright Fluentd Authors` instead?
(in accordance with the discussion we've been having for OpenTracing: https://github.com/opentracing/opentracing-go/pull/181#issuecomment-376668544)

Comment (edsiper) [reply LICENSE]: LGTM "Copyright Fluentd Authors" :)

Comment (kzk) [reply LICENSE]: Should we mass-change the Copyright in the source code as well?

Comment (dankohn) [reply LICENSE]: @kzk Feel free, but it's not necessary.

**Comment by eirinikos** [comment]
I've updated `LICENSE` and `README` to refer to `Copyright 2011-2018 Fluentd Authors`.

I've also add my DCO sign-off (which was previously missing).

If there are no more comments, this PR is ready to be merged by a maintainer! :)

**Comment by repeatedly** [comment]
Thanks! Just merged :)

**Pull request merged**